libs/sdk-core/src/breez_services.rs

```diff
@@ -2129,6 +2130,40 @@ impl Receiver for PaymentReceiver {
             info!("Signed invoice with hint = {}", signed_invoice_with_hint);
 
             parsed_invoice = parse_invoice(&signed_invoice_with_hint)?;
+
+            self.persister.insert_or_update_payments(
```

Comment:
We already have a point where we use `insert_open_channel_payment_info` to add the payer invoice amount.
I think this method call can be changed to contain the payer invoice instead of calling `insert_or_update_payments`